### PR TITLE
Added checkboxes for vaccination and had corona

### PR DESCRIPTION
**Pull request by bitboxer**
closes https://github.com/railslove/recover-backlog/issues/144, https://github.com/railslove/recover-backlog/issues/143

### CODE REVIEW

**Comment by bitboxer** [comment]
Screen Shot 2021-05-19 at 17 49 14

**Review by salimhb — APPROVED** [review]

**Comment by bitboxer** [comment]
We cannot merge this just yet, because @koos changed the specification in the ticket a bit. Need to clearify before continuing

**Comment by bitboxer** [comment]
This is now changed a bit. It stores the type a user clicks in the ticket and adds it to the export.

File: ui/blocks/Onboarding.tsx [hunk 1]
```diff
@@ -94,9 +95,22 @@ export const Onboarding: React.FC<OnboardingProps> = ({
           {area.companyNeedToShowCoronaTest && (
             <>
               <Box height={3} />
-              <Checkbox
-                name="haveNegativeTest"
-                label="Ich bestätige einen negativen Test vorliegen zu haben"
+              <Radio
+                name="providedHealthDocument"
+                label="Getestet: Ich bestätige ein negatives, nicht älter als 48 Stunden zurückliegendes, Testergebnis vorliegen zu haben und dieses im Prüfall vorweisen zu können."
```
Comment (bonflintstone) [inline ui/blocks/Onboarding.tsx:100]:
```suggestion
                label="Getestet: Ich bestätige ein negatives, nicht länger als 48 Stunden zurückliegendes, Testergebnis vorliegen zu haben und dieses im Prüffall vorweisen zu können."
```

**Review by bonflintstone — APPROVED** [review]

**Pull request merged**